libsql-server/src/main.rs

```diff
@@ -332,12 +332,19 @@ fn enable_libsql_logging() {
 }
 
 fn make_db_config(config: &Cli) -> anyhow::Result<DbConfig> {
+    let mut bottomless_replication = config
+        .enable_bottomless_replication
+        .then(bottomless::replicator::Options::from_env)
+        .transpose()?;
+    // Inherit encryption key for bottomless from the db config, if not specified.
+    if let Some(ref mut bottomless_replication) = bottomless_replication {
+        if bottomless_replication.encryption_key.is_none() {
```

Comment:
The question would be: if frame headers are not encrypted, and page_size+changecounter are not read straight from DB file, does bottomless even need to know that pages are encrypted? IIRC there's no place where it does read a page contents outside of the WAL checksum verification.

Reply:
once page_size and changecounter are not read straight from the db file, bottomless can be encryption-agnostic, yes